src/roswire/ros2/state.py

```diff
@@ -116,6 +125,11 @@ def probe(self) -> ROS2SystemState:
             _ACTION_SERVERS: {},
             _ACTION_CLIENTS: {}
         }
+        # The place to store type information
```

Comment:
I don't think that the comment is really necessary here

Reply:
OK.